plasma_framework/migrations/1050_output_results.js

```diff
@@ -0,0 +1,52 @@
+/* eslint-disable no-console */
+
+const fs = require('fs');
+const path = require('path');
+const config = require('../config.js');
+
+const PlasmaFramework = artifacts.require('PlasmaFramework');
+const PaymentEip712LibMock = artifacts.require('PaymentEip712LibMock');
+const MerkleWrapper = artifacts.require('MerkleWrapper');
+const ERC20Mintable = artifacts.require('ERC20Mintable');
+module.exports = async (
+    deployer,
+    _,
+    // eslint-disable-next-line no-unused-vars
+    [deployerAddress, maintainerAddress, authorityAddress],
+) => {
+    const vault = process.env.VAULT || false;
+    if (!vault) {
```

Comment:
Maybe output doesn't show up for multisig?

Reply:
yeah until the multisig actually executed all pending transactions we would have faulty outputs.

Reply:
Can we remove the vault check here? vault or no vault still just getting the deployed instances i mean 🤔

Reply:
I think these three:
```
const ethVault = await plasmaFramework.vaults(config.registerKeys.vaultId.eth);
        const erc20Vault = await plasmaFramework.vaults(config.registerKeys.vaultId.erc20);
        const paymentExitGame = await plasmaFramework.exitGames(config.registerKeys.txTypes.payment);
```
would return faulty results since vaults aren't registered and exit games as well. is that what you meant?

Reply:
umm migration 1040 has register vault and exit game right?

Reply:
Maybe these values https://github.com/omgnetwork/plasma-contracts/pull/678/files#diff-a9446999f888af4dc205cab368614004R34

Reply:
```
1050_output_results.js
======================
{"authority_address":"0x7633fe8542c2218b5a25777477f63d395aa5afb4","eth_vault":"0x0000000000000000000000000000000000000000","erc20_vault":"0x0000000000000000000000000000000000000000","payment_exit_game":"0x0000000000000000000000000000000000000000","plasma_framework_tx_hash":"0x6fdc1e60ba970127239da7c3aa44121cd1a5f44e2a135434e686d44e38026c6b","plasma_framework":"0x2d697f6fb489326ecaabce7fc67678d3e8b838b1"}

- Saving migration to chain.
   > Saving migration to chain.
   -------------------------------------
   > Total cost:    
```
That's because https://github.com/omgnetwork/plasma-contracts/pull/678/files#diff-a9446999f888af4dc205cab368614004R54-R56 only puts the transactions into the multisig queue, not execute them. So techically they're waiting for the multisignature accounts to approve and call execute!

Reply:
I mean yeah, I can leave it as is:
```
"eth_vault":"0x0000000000000000000000000000000000000000"
"erc20_vault":"0x0000000000000000000000000000000000000000"
"payment_exit_game":"0x0000000000000000000000000000000000000000"
```

Doesn't harm anyone 👍

Reply:
Aah i see. Let's see what we get here, if there is a problem we could probably just have whatever addresses are there logged? wdyt?

Reply:
awesome! we got what you said! 🚀